apps/actors/lib/actors/actor/entity/entity.ex

```diff
@@ -290,14 +324,34 @@ defmodule Actors.Actor.Entity do
     |> parse_packed_response()
   end
 
+  defp do_handle_info(
+         {:invoke_timer_command, %Command{name: cmd} = _command},
+         %EntityState{
+           system: _actor_system,
+           actor: actor
+         } = state
+       ) do
+    invocation = %InvocationRequest{
+      actor: actor,
+      command_name: cmd,
+      value: Any.new(),
+      async: true
+    }
+
+    do_handle_cast({:invocation_request, invocation}, state)
```

Comment:
shouldnt we get the state from `:invocation_request` here?

Reply:
Yeah this bypass all genserver related stuff but preserves all consistency related to command execution

Reply:
No, state that would be passed there is the same as this one. Actors have the same behavior as a single consumer queue, we are inside the handle_info callback the handle_info state is the global state of the actor no matter which callback is being invoked. Callbacks run sequentially so it's safe to pass state here.